src/components/basic-element/types/link.jsx

```diff
@@ -26,6 +26,11 @@ var spec = new Spec('link', assign({
     validation: React.PropTypes.string,
     default: ''
   },
+  targetWebURL: {
+    category: 'attributes',
+    validation: React.PropTypes.string,
```

Comment:
Hmm. Possibly. That would only be helpful if the incoming validation code for the URL input regresses and then this would serve as a warning to devs. We'd have to share validation between both points to keep it DRY though, which would mean externalizing validation for URLs, which seems like more trouble than it's worth right now.

It might make sense to do a follow up ticket for adding a general purpose validation library though...